docs/versioned_docs/version-v1.2.0/the_aztec_network/index.md

```diff
@@ -1,62 +1,119 @@
 ---
 id: index
 sidebar_position: 0
-title: The Aztec Network
+title: Running a full node
 ---
 
-In this section, you can explore how governance works on Aztec, the types of nodes that can be run, and how to contribute to decentralization on the Aztec Testnet.
-
-## Learn about governance on Aztec
-
-<div className="card-container full-width">
-  <Card shadow='tl' link='the_aztec_network/concepts/governance'>
-    <CardHeader>
-      <h3>Governance</h3>
-    </CardHeader>
-    <CardBody>
-        An introduction to Aztec's governance model
-    </CardBody>
-  </Card>
-</div>
-
-<!-- ## Decentralization Components
-
-<div className="card-container">
-  <Card shadow='tl' link='the_aztec_network/concepts/provers-and-sequencers/index'>
-    <CardHeader>
-      <h3>Sequencers</h3>
-    </CardHeader>
-    <CardBody>
-      How sequencers propose and produce blocks
-    </CardBody>
-  </Card>
-</div> -->
-
-## Run a node
-
-<div className="card-container">
-  <Card shadow='tl' link='the_aztec_network/guides/run_nodes/how_to_run_sequencer'>
-    <CardHeader>
-      <h3>Run a validator node (sequencer)</h3>
-    </CardHeader>
-
-  </Card>
-
-  <Card shadow='tl' link='the_aztec_network/guides/run_nodes/how_to_run_prover'>
-    <CardHeader>
-      <h3>Run a prover node</h3>
-    </CardHeader>
-
-  </Card>
-</div>
-
-## FAQs
-
-<div className="card-container">
-  <Card shadow='tl' link='the_aztec_network/guides/run_nodes/operator_faq'>
-    <CardHeader>
-      <h3>FAQs & Common Issues/Resolutions</h3>
-    </CardHeader>
-
-  </Card>
-</div>
+This guide will go over the steps required to run a full node on Aztec with a basic setup.
+
+A full node allows users to connect and interact with the network. It provides users an interface to send and receive transactions and state updates without relying on a third party.
+
+A user should run their own full node if they want to interact with the network in the most privacy preserving way. Furthermore, it is a great way to support the Aztec network and get involved with the community.
+
+Please note that there are two other types of nodes available that we are not covering in this guide. One is a sequencer node, and the other is a prover node. For more information on the other types of nodes and how they act in the protocol, please see here [link].
+
+Furthermore, there are advanced guides that can be found here [link] that go into more feature-complete setups that are tailored for more complex use cases.
+
+Prerequisities:
+Minimum hardware requirements
```

Comment:
Add a line break bc it currently looks like this 
<img width="780" height="96" alt="image" src="https://github.com/user-attachments/assets/1037d9b5-f6ef-4597-8cd2-aa84657237f4" />


```suggestion

Minimum hardware requirements:
```